### PR TITLE
[CMake] Add plutil to CMake build

### PR DESCRIPTION
**Pull request by jmschonfeld**
This restores `plutil` to the new CMake build

### CODE REVIEW

File: Sources/plutil/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+target_link_options(plutil PRIVATE
+    "SHELL:-no-toolchain-stdlib-rpath")
+
+set_target_properties(plutil PROPERTIES
+	INSTALL_RPATH "$ORIGIN/../lib/swift/${SWIFT_SYSTEM_NAME}")
+
+set_property(GLOBAL APPEND PROPERTY Foundation_EXPORTS plutil)
+install(TARGETS plutil
+	DESTINATION ${CMAKE_INSTALL_BINDIR})
```
Comment (iCharlesHu) [inline Sources/plutil/CMakeLists.txt:15]: (nit) missing new line

File: Sources/plutil/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+add_executable(plutil
```
Comment (iCharlesHu) [inline Sources/plutil/CMakeLists.txt:1]: Missing file header here.

**Review by iCharlesHu — APPROVED** [review]

File: Sources/plutil/CMakeLists.txt (hunk 2 above)
Comment (jmschonfeld) [reply Sources/plutil/CMakeLists.txt:15]: Fixed in 97442722cc9aa16a7d6961a86fc2d6e26a3493ad

File: Sources/plutil/CMakeLists.txt (hunk 1 above)
Comment (jmschonfeld) [reply Sources/plutil/CMakeLists.txt]: Fixed in 97442722cc9aa16a7d6961a86fc2d6e26a3493ad

**Pull request merged**